### PR TITLE
Change Zoo Image Preprocessing name

### PR DESCRIPTION
**Pull request by jenniew**
Rename all image preprocessing to ImageXXX to prevent potential errors, e.g., ImageCenterCrop, etc.

### CODE REVIEW

**Comment by jenniew** [comment]
Related issue: https://github.com/intel-analytics/zoo/issues/232

**Comment by jason-dai** [comment]
How about apps?

**Comment by jason-dai** [comment]
Is this ready for merge?

**Comment by jenniew** [comment]
Already update the apps

File: pyzoo/zoo/examples/nnframes/imageTransferLearning/ImageTransferLearningExample.py [hunk 1]
```diff
@@ -55,8 +55,8 @@
 
     # compose a pipeline that includes feature transform, pretrained model and Logistic Regression
     transformer = ChainedPreprocessing(
-        [RowToImageFeature(), Resize(256, 256), CenterCrop(224, 224),
-         ChannelNormalize(123.0, 117.0, 104.0), MatToTensor(), ImageFeatureToTensor()])
+        [RowToImageFeature(), ImageResize(256, 256), ImageCenterCrop(224, 224),
+         ChannelNormalize(123.0, 117.0, 104.0), ImageMatToTensor(), ImageFeatureToTensor()])
```
Comment (jason-dai) [inline pyzoo/zoo/examples/nnframes/imageTransferLearning/ImageTransferLearningExample.py]: how about ChannelNormalize?

File: pyzoo/zoo/examples/nnframes/imageInference/ImageInferenceExample.py [hunk 2]
```diff
@@ -31,8 +31,8 @@ def inference(image_path, model_path, sc):
     imageDF = NNImageReader.readImages(image_path, sc)
     getName = udf(lambda row: row[0], StringType())
     transformer = ChainedPreprocessing(
-        [RowToImageFeature(), Resize(256, 256), CenterCrop(224, 224),
-         ChannelNormalize(123.0, 117.0, 104.0), MatToTensor(), ImageFeatureToTensor()])
+        [RowToImageFeature(), ImageResize(256, 256), ImageCenterCrop(224, 224),
+         ChannelNormalize(123.0, 117.0, 104.0), ImageMatToTensor(), ImageFeatureToTensor()])
```
Comment (jason-dai) [inline pyzoo/zoo/examples/nnframes/imageInference/ImageInferenceExample.py]: how about ChannelNormalize?

File: pyzoo/zoo/examples/nnframes/finetune/transfer_learning.py [hunk 3]
```diff
@@ -56,8 +56,8 @@
 
     # compose a pipeline that includes feature transform, pretrained model and Logistic Regression
     transformer = ChainedPreprocessing(
-        [RowToImageFeature(), Resize(256, 256), CenterCrop(224, 224),
-         ChannelNormalize(123.0, 117.0, 104.0), MatToTensor(), ImageFeatureToTensor()])
+        [RowToImageFeature(), ImageResize(256, 256), ImageCenterCrop(224, 224),
+         ChannelNormalize(123.0, 117.0, 104.0), ImageMatToTensor(), ImageFeatureToTensor()])
```
Comment (jason-dai) [inline pyzoo/zoo/examples/nnframes/finetune/transfer_learning.py]: how about ChannelNormalize?

**Comment by jason-dai** [comment]
Some minor comments; otherwise it looks good to me.

File: pyzoo/zoo/examples/nnframes/imageTransferLearning/ImageTransferLearningExample.py (hunk 1 above)
Comment (jenniew) [reply pyzoo/zoo/examples/nnframes/imageTransferLearning/ImageTransferLearningExample.py]: Updated

File: pyzoo/zoo/examples/nnframes/imageInference/ImageInferenceExample.py (hunk 2 above)
Comment (jenniew) [reply pyzoo/zoo/examples/nnframes/imageInference/ImageInferenceExample.py]: Updated

File: pyzoo/zoo/examples/nnframes/finetune/transfer_learning.py (hunk 3 above)
Comment (jenniew) [reply pyzoo/zoo/examples/nnframes/finetune/transfer_learning.py]: Updated

File: docs/docs/models/imageclassification/README.md [hunk 4]
```diff
@@ -16,15 +20,45 @@ val data = ImageSet.read(params.image, sc, params.nPartition)
 val output = model.predictImageSet(data)
 ```
 
+User can also define his own configuration to do the inference with below code piece.
 
-[Python example](https://github.com/intel-analytics/zoo/tree/master/pyzoo/zoo/examples/imageclassification)
+```scala
+val model = ImageClassifier.loadModel[Float](params.model)
+val data = ImageSet.read(params.image, sc, params.nPartition)
+val featureTransformer = Resize(256, 256)-> CenterCrop(224, 224) ->
```
Comment (jason-dai) [inline docs/docs/models/imageclassification/README.md]: preprocessing = ImageResize() -> ...

File: docs/docs/models/imageclassification/README.md [hunk 5]
```diff
@@ -16,15 +20,45 @@ val data = ImageSet.read(params.image, sc, params.nPartition)
...
 model = ImageClassifier.load_model(model_path)
 image_set = ImageSet.read(img_path, sc)
 output = imc.predict_image_set(image_set)
 ```
+
+User can also define his own configuration to do the inference with below code piece.
+```
+model = ImageClassifier.load_model(model_path)
+image_set = ImageSet.read(img_path, sc)
+transformer = Pipeline([Resize(256,256),
```
Comment (jason-dai) [inline docs/docs/models/imageclassification/README.md]: Pipeline does not work for preprocessing

Comment (jenniew) [reply docs/docs/models/imageclassification/README.md]: remove it. and ChainedPreprocessing does not work in  python. I'll file a issue

File: docs/docs/models/imageclassification/README.md (hunk 4 above)
Comment (jenniew) [reply docs/docs/models/imageclassification/README.md]: upadated

**Comment by jason-dai** [comment]
LGTM

**Comment by jenniew** [comment]
Jenkins passed: http://172.168.2.101:8080/view/ZOO-PR/job/ZOO-PR-Validation/421/

**Pull request merged**